README.md

```diff
@@ -1,40 +1,46 @@
-# cowin-notifier
 
-A cross platform tool which instantly notifies about COVID vaccine availability.
+
+# 💉 CoWIN Notifier 😷
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/aryan9600/cowin-notifier)
+[![GitHub License](https://img.shields.io/github/license/aryan9600/cowin-notifier)](https://github.com/aryan9600/cowin-notifier/blob/master/LICENSE)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/aryan9600/cowin-notifier/issues/new/choose)
+![release](https://img.shields.io/github/v/release/aryan9600/cowin-notifier)
+![rust-version](https://img.shields.io/badge/rust-v1.5.2-red)
+
+A cross-platform tool written in rust, which instantly notifies users about COVID-19 vaccine availability at their regions.
 
 ## Usage
 
-### Windows
-* [Downlaod the zip file](https://github.com/aryan9600/cowin-notifier/releases/download/v0.1/cowin-notifier-windows.zip) and unzip it.
+### 💻 Windows
+* [Download the zip file](https://github.com/aryan9600/cowin-notifier/releases/download/v0.1/cowin-notifier-windows.zip) and unzip it.
 * Open the folder containing the file, do a `Shift + Right Click` and select "Open Powershell here."
-* Run `./cowin-notifier "state_name", "district_name", "age_group"`
-![Alt text](https://raw.githubusercontent.com/aryan9600/cowin-notifier/main/assets/WhatsApp%20Image%202021-05-06%20at%2021.23.49.jpeg)
+* Run `./cowin-notifier "state_name" "district_name" "age_group"`
+![Windows Screenshot](https://raw.githubusercontent.com/aryan9600/cowin-notifier/main/assets/WhatsApp%20Image%202021-05-06%20at%2021.23.49.jpeg)
 * Minimize the Powershell window.
 * If a slot opens up for your age group and district, you'll recieve a notification. __Make sure that you don't close the Powershell window.__
 
-### macOS
+### 🍎 macOS
 * [Download the zip file](https://github.com/aryan9600/cowin-notifier/releases/download/v0.1/cowin-notifier-macos.zip) and unzip it.
-* Open your terminal and `cd` into your default Downloads directory. macOS users can fire up Spotlight(cmd+Space), search for Terminal and open it.
-
-![Alt text](https://raw.githubusercontent.com/aryan9600/cowin-notifier/main/assets/Screen%20Shot%202021-05-05%20at%2015.59.44.png)
-
-* Run `cd ~/Downloads`
-
-* Run `./cowin-notifier "state_name", "district_name", "age_group"`
-![Alt text](https://raw.githubusercontent.com/aryan9600/cowin-notifier/main/assets/Screen%20Shot%202021-05-05%20at%2016.06.28.png)
-
+* Open your terminal and `cd` into your default Downloads directory where you have downloaded the file. macOS users can fire up Spotlight using `cmd + Space`, search for Terminal and open it.
+![macOS Screenshot-1](https://github.com/mintbomb27/cowin-notifier/raw/main/assets/macOS%20spotlight.png)
+* Run `cd ~/Downloads` 
+* Run `chmod +x cowin-notifier`
+* Run `./cowin-notifier "state_name" "district_name" "age_group"`
```

Comment:
I think it would look a bit better if they are block commands and eliminate the redundant `Run` word in each step, something similar to:

* Run the following commands:
```
cd ~/Downloads
chmod +x cowin-notifier
./cowin-notifier "state_name" "district_name" "age_group"
```

Reply:
Also, don't think you need to `chmod` the binary.

Reply:
I actually tried at first without the chmod, but didn't work. That is why I added it.

Reply:
@aryan9600 I believe some unzippers preserve the state permission and have safe default permission set to 666 or 644. Hence it's a safe option to` chmod +x` it. However, if #4 gets merged. This step wouldn't be required anymore and we can also shift to a bash script that places the binary in user's `bin` folder.